static/js/components/widgets/ResourcePickerListing.test.tsx

```diff
@@ -51,12 +56,12 @@ describe("ResourcePickerListing", () => {
     // @ts-ignore
     useWebsite.mockReturnValue(website)
 
-    contentListingItems = [
-      [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
-      [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
-      [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
-      [makeWebsiteContentDetail(), makeWebsiteContentDetail()]
-    ]
+    contentListingItems = {
+      videos:      [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
+      documents:   [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
+      pages:       [makeWebsiteContentDetail(), makeWebsiteContentDetail()],
+      courseLists: [makeWebsiteContentDetail(), makeWebsiteContentDetail()]
+    }
```

Comment:
Almost all the changes in this file are related to this renaming, because trying to remember which index of `contentListingItems` was which thing was hard...